packages/gatsby-transformer-sharp/src/customize-schema.js

```diff
@@ -504,9 +504,8 @@ const imageNodeType = ({
         type: TransformOptionsType,
         description: `Options to pass to sharp to control cropping and other image manipulations.`,
       },
-      background: {
+      backgroundColor: {
```

Comment:
Did we have any other doc where this needs to be changed? I don't think so, but wanted to mention.

Reply:
No, I don't think we mention it